README.md

```diff
@@ -8,15 +8,24 @@ Each preconfigured solution implements a common IoT scenario and is a complete,
 ##Remote Monitoring preconfigured solution
 The remote monitoring preconfigured solution illustrates how you can perform end-to-end monitoring. It brings together key Azure IoT services to enable the following features: data ingestion, device identiy, command and control, rules and actions.
 
-#### Wiki: 
-* Do you want to know how to deploy this preconfigured solution locally and to the cloud? Take a look at our [Wiki](https://github.com/Azure/azure-iot-remote-monitoring/wiki) for answers.
-* Have ideas for how we can improve Azure IoT? Give us [Feedback](http://feedback.azure.com/forums/321918-azure-iot).
-
 ##Contents of this repository
-#### Web folder:
+
+### Docs folder:
+  * [Set up development environment (Windows)](Docs/dev-setup.md) outlines the prerequisites for deploying the remote monitoring preconfigured solution.
+  * [Local deployment and debugging](Docs/local-deployment.md) describes how to deploy locally and basic debugging.
+  * [Cloud deployment](Docs/cloud-deployment.md) describes building and deploying the remote monitoring preconfigured solution fully on Azure.
+  * [Add co-administrators](Docs/add-coadmins.md) on your Azure subscription (recommended).
+  * [Manually setting up roles and assigning permissions in Azure Active Directory (AAD)](Docs/roles-and-permissions.md).
+  
+Other useful [IoT Suite documentation](https://azure.microsoft.com/documentation/suites/iot-suite/):
+  * [Frequently asked questions for IoT Suite](https://azure.microsoft.com/documentation/articles/iot-suite-faq/)
+  * [Permissions on the azureiotsuite.com site](https://azure.microsoft.com/documentation/articles/iot-suite-permissions/)
+  
+  
+### Web folder:
```

Comment:
Remove web folder, infrastructure folder, and simulator folder. The solution structure has been refactored and these folders no longer align.